packages/react-core/src/components/ExpandableSection/examples/ExpandableSection.md

```diff
@@ -178,3 +180,48 @@ class SimpleExpandableSection extends React.Component {
   }
 }
 ```
+
+### With custom component
```

Comment:
I would rename this to something like "With icon in toggle".  What do you think for a name here @mcarrano. 
I think we want to get across the the consumer that the toggle can be something other than a string.

Reply:
Maybe say "With additional non-text elements?" We shouldn't imply that this is limited to an icon. I'd also recommend that we note that this is limited to non-interactive elements (you can't put a button here, for example).